<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SafeWindow</class>
 <widget class="QMainWindow" name="SafeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1306</width>
    <height>734</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1306</width>
    <height>734</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1306</width>
    <height>734</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GameOfSaves</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>598</x>
      <y>550</y>
      <width>110</width>
      <height>112</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>112</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>112</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>400</width>
      <height>460</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>460</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>460</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Safe/Assets/safe_game.png);
background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>American Typewriter</family>
      <pointsize>35</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 35pt &quot;American Typewriter&quot;;
color: #000000;
border: 2px solid black;
border-radius: 4px;
padding: -1px;
background: #FFFFFF;
</string>
    </property>
    <property name="text">
     <string>_ _ _</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>400</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Safe/Assets/bet350.png);
background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>553</x>
      <y>320</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 4px;
padding: -1px;
background: #FFFFFF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>430</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #000000;
border: 2px solid black;
border-radius: 4px;
padding: -1px;
background: #FFFFDD;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>430</y>
      <width>41</width>
      <height>121</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #000000;
border: 2px solid black;
border-radius: 4px;
padding: -1px;
background: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>В
В
О
Д</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>450</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #000000;
border: 2px solid black;
border-radius: 4px;
padding: -1px;
background: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>480</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #000000;
border: 2px solid black;
border-radius: 4px;
padding: -1px;
background: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>510</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #000000;
border: 2px solid black;
border-radius: 4px;
padding: -1px;
background: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>480</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #000000;
border: 2px solid black;
border-radius: 4px;
padding: -1px;
background: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>510</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #000000;
border: 2px solid black;
border-radius: 4px;
padding: -1px;
background: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>450</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #000000;
border: 2px solid black;
border-radius: 4px;
padding: -1px;
background: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>480</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #000000;
border: 2px solid black;
border-radius: 4px;
padding: -1px;
background: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>510</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #000000;
border: 2px solid black;
border-radius: 4px;
padding: -1px;
background: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>450</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #000000;
border: 2px solid black;
border-radius: 4px;
padding: -1px;
background: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>533</x>
      <y>430</y>
      <width>240</width>
      <height>73</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>73</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>73</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>150</y>
      <width>391</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Money/Assets/console.jpg);
color: #FF7266;
font: 20pt &quot;Papyrus&quot;;
border: 2px solid red;
border-radius: 4px;
padding: -1px;</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>553</x>
      <y>320</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Safe/Assets/tries3.png);
background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1306</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
